src/resizable.directive.ts

```diff
@@ -374,6 +377,7 @@ export class ResizableDirective implements OnInit, OnChanges, OnDestroy {
    * @hidden
    */
   constructor(
+    @Inject(PLATFORM_ID) private platformId: object,
```

Comment:
I think this is causing the build to fail, if you set the type to `any` it might fix it